core/types/index.d.ts

```diff
@@ -1,9 +1,110 @@
-interface IStorageBackend {
-  getSingle(): Promise<any>;
-  //todo
+/**
+ *
+ * Core Async Storage API
+ *
+ */
+declare class AsyncStorage<
```

Comment:
can't these types be output by tsc?

Reply:
They were at first, but then I decided to remove it, as I needed more control over it